### ISSUE TITLE
For whatever Reason, WQAchievements prevents ElvUI from hiding theOrderHallCommandBar from Legion!

### ISSUE THREAD

**Issue by fubaWoW** [Urtgard/WQAchievements#149]
hi,
i don't know why and what you're doing but with WQAchievements active it prevents ElvUI from Hide the OrderHallCommandBar!

**Comment by Urtgard** [comment]
Sounds weird. Are you sure it's because of WQA?
Do you get an error message?

**Comment by fubaWoW** [comment]
>Are you sure it's because of WQA?
Yes, tested a couple times with just ElvUI and WQA active and just happend at the Order Hall

>Do you get an error message?
No, there was no LUA Error but the Frame OrderHallCommandBar was forced to Show even it was hidden by ElvUI

**Comment by Urtgard** [comment]
When you change the option to show or hide the bar and close the option menu, ElvUI asks you to reload the ui. Maybe you missed that?
